ground/src/main/res/values/dimens.xml

```diff
@@ -39,4 +39,16 @@
   <dimen name="job_card_corner_radius">16dp</dimen>
   <dimen name="job_card_start_end_margin">8dp</dimen>
   <dimen name="zoom_on_cluster_click_padding">50dp</dimen>
+
+  <!-- Standard Spacing -->
+  <dimen name="spacing_small">4dp</dimen>
+  <dimen name="spacing_medium">8dp</dimen>
+  <dimen name="spacing_large">16dp</dimen>
+  <dimen name="spacing_xlarge">24dp</dimen>
+
+  <!-- Text Sizes -->
+  <dimen name="text_small">12sp</dimen>
+  <dimen name="text_medium">14sp</dimen>
+  <dimen name="text_large">16sp</dimen>
+  <dimen name="text_xlarge">20sp</dimen>
```

Comment:
I don't think the intention is to add arbitrary dimens to this file. But rather to move the hardcoded values to this file. However, I feel we should not spend too much time on this since Compose UI doesn't use this file. Do you know what is the recommended approach for compose UI?

Reply:
As per documentation 
`dimensionResource(R.dimen.padding_small)`

Hence using dimens.xml

Same as what we are doing for strings

Reply:
wdyt about naming?

Reply:
LGTM